src/modules/workflows/workflows.service.ts

```diff
@@ -110,7 +110,8 @@ export class WorkflowsService {
     let failed = false;
 
     for (const reviewTask of clinicalReviewTasks) {
-      const roles = reviewTask?.args['reviewer_roles'];
+      const commaSeparatedRoles = reviewTask?.args['reviewer_roles'];
+      const roles: string[] = commaSeparatedRoles.split(',');
```

Comment:
there could potentially be an insatnce wehre `commaSeparatedRoles` is `undefined`. we probably should do something like this

```suggestion
      const roles: string[] = commaSeparatedRoles?.split(',') ?? [];
```